docs/src/hooks/use-site-metadata.js

```diff
@@ -2,7 +2,7 @@ import { useStaticQuery, graphql } from 'gatsby';
 
 function useSiteMetadata() {
 	const data = useStaticQuery(graphql`
-		{
+		query {
```

Comment:
Why is this changing from before? i.e. how did it work previously without the query???

Reply:
I changed it for consistency between the two queries, and confirmed the change doesn't affect anything.